kube-client/src/api/util/mod.rs

```diff
@@ -38,6 +41,24 @@ impl Api<Node> {
     }
 }
 
+impl Api<ServiceAccount> {
+    /// Create a TokenRequest of a ServiceAccount
+    pub async fn create_token_request(
```

Comment:
I __think__ this can also have an integration test creating a tokenrequest against the `kubernetes` service in the default namespace:

```rust
#[tokio::test]
#[ignore] # requires a cluster
fn portforward_path() -> Result<()> {
    use k8s_openapi::api::core::v1::ServiceAccount;
    use kube::{Client, Api, api::PostParams};
    let client = Client::try_default().await?;
    let api: Api<ServiceAccount> = Api::namespaced(client, "default");
    let pp = PostParams::default();
    let token = TokenRequest { /* not sure what is sensible here */ };
    api.create_token_request("kubernetes", pp, token).await?;
    Ok(())
}
```